lib/utils.js

```diff
@@ -178,31 +178,43 @@ function endOfMonthTimestamp() {
   return Date.UTC(curDate.getUTCFullYear(), curDate.getUTCMonth() + 1, 1, 0, 0, 0, 0);
 }
 
-function resolveElevatedParams(erlParams, bucketKeyConfig) {
+function resolveElevatedParams(erlParams, bucketKeyConfig, key) {
   // provide default values for elevated_limits unless the bucketKeyConfig has them
-  return {
+  const elevatedLimits = {
     ms_per_interval: bucketKeyConfig.ms_per_interval,
     size: bucketKeyConfig.size,
     erl_activation_period_seconds: 0,
     erl_quota: 0,
     erl_quota_interval: 'quota_per_calendar_month',
-    erl_is_active_key: 'defaultActiveKey',
-    erl_quota_key: 'defaultQuotaKey',
+    erl_is_active_key: 'ERLActiveKey',
```

Comment:
Can we inline these?

```suggestion
    erl_is_active_key: replicateHashtag(key, 'ERLActiveKey')
    erl_quota_key: replicateHashtag(key, 'ERLQuotaKey')
```